app/models/book.rb

```diff
@@ -8,4 +8,13 @@ class Book < ApplicationRecord
   def long?
     number_of_pages > 500
   end
+
+  def contributors
+    contributions.includes(:contributor).map(&:contributor)
```

Comment:
Modify this method so it gives back the info filtered out using the json.contributors block in the jbuilder views